Text/PrettyPrint/Leijen/Text/Monadic.hs

```diff
@@ -96,24 +96,24 @@ instance Monad m => IsString (m Doc) where
 --   horizontally if that fits the page. Otherwise they are aligned
 --   vertically. All comma separators are put in front of the
 --   elements.
-list :: (Monad m) => m [Doc] -> m Doc
```

Comment:
I prefer parentheses even for singleton constraints (so they don't have to get added when another constraint is added).